.github/actions/build_pandas/action.yml

```diff
@@ -14,6 +14,25 @@ runs:
         micromamba list
       shell: bash -el {0}
 
+    - name: Make ccache-action key
+      id: ccache-action-key
+      run: |
+        key="${{ runner.os }}--${{ runner.arch }}--${{ github.workflow }}"
+        # Date: Daily invalidation of all ccaches as an extra safety measure.
+        key="$key--$(/bin/date -u '+%Y%m%d')"
```

Comment:
Not sure if this has any advantages, I think it was requested as a safety measure by some reviewer in one of the earlier versions of this PR. Not sure if it's necessary since caches are automatically rotated out anyways when the cache is full